docs-new/docs/graphql/core/migrations/manage-metadata.mdx

```diff
@@ -115,47 +94,30 @@ instance's metadata onto itself.
 Importing completely replaces the metadata on that instance, i.e. you
 lose any metadata that was already present before.
 
-<div class="rst-class">
-
-api_tabs
-
-</div>
-
-<div class="tabs">
-
-<div class="tab">
-
-CLI
-
-Metadata can be applied with the
-`hasura metadata apply <hasura_metadata_apply>` command.
-
-</div>
-
-<div class="tab">
+<Tabs className="api-tabs">
+<TabItem value="cli" label="CLI">
+Metadata can be applied with the <Link to="/graphql/core/hasura-cli/hasura_metadata_apply">
+hasura metadata apply
+</Link> command.
+</TabItem>
```

Comment:
Pls add an empty line before and after and starting the content from line beginning will help avoid using Link compinent.

The broken links with Link component wont be caught on build. SO lets use markdown links where ever possible.